addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/BigRedirectsScanRule.java

```diff
@@ -111,7 +111,7 @@ private AlertBuilder createBaseAlert(String ref) {
         return newAlert()
                 .setRisk(Alert.RISK_LOW)
                 .setConfidence(Alert.CONFIDENCE_MEDIUM)
-                .setSolution(getSolution())
+                .setSolution(Constant.messages.getString(MESSAGE_PREFIX + "soln"))
```

Comment:
This will mean we lose the solution from the alert pages.

Reply:
This one does have an example alert.